mirrord/sip/src/lib.rs

```diff
@@ -503,7 +506,19 @@ mod main {
     /// SIP check for binaries.
     fn is_binary_sip(path: &Path, data: &[u8], opts: SipPatchOptions) -> Result<bool> {
         // Skip patching binary if it is in the list of binaries to skip.
-        if opts.skip.iter().any(|x| path.ends_with(x)) {
+        if opts
+            .skip
+            .as_ref()
+            .unwrap_or(
+                &MIRRORD_SIP_DEFAULT_SKIP_LIST
```

Comment:
maybe this is a nitpick: maybe it's better to add the default value directly in the configuration, instead of where using that configuration. There could be more usages of the `skip_sip` config field, in the future, and it's easier to stay consistent if we define it at the source. What do you think?

Reply:
The only concern I had was that I want to keep the default list in the sip crate. Add it as the default value in the config crate will make it depend on the sip crate. But, yeah, keeping it in one place is less error-prone. I can move the `const` list to the config crate.